src/rebuild/srv.c

```diff
@@ -1093,7 +1093,7 @@ rebuild_debug_print_queue()
 	 * This only accumulates the targets in a single task, so it doesn't
 	 * need to be very big. 200 bytes is enough for ~30 5-digit target ids
 	 */
-	char tgts_buf[200];
+	char tgts_buf[200] = { 0 };
```

Comment:
It would be slightly better to put this on L1103 as it reduces the scope of the variable and would re-set it for each loop.

Reply:
sure